draft-ietf-ppm-dap.md

```diff
@@ -2224,10 +2229,6 @@ input share in the job:
 1. Check that the input share can be decoded as specified by the VDAF. If not,
    the input share MUST be marked as invalid with the error `invalid_message`.
 
-1. Check that the report's timestamp is well-formed as specified in
```

Comment:
I just want to check my own understanding here. We've removing this part of the algorithm because the command `the Aggregator MUST mark the input share as invalid with error 'invalid_message'` is incorrect. 

In contrast to the next three checks, we actually want a malformed timestamp _in any report_ to cause the whole `AggregationJobInitReq` to be rejected, not the individual report.

Reply:
Yeah: the other checks are errors that could be caused by an individual client that's buggy or otherwise defective, and we don't want that to spoil the entire aggregation job, much as we don't want a single invalid proof to spoil the rest of the job.

However, an aggregation job containing a malformed timestamp indicates a defect in the leader, which is either inserting bad timestamps when it constructs aggregation jobs or failing to weed them out when it handles uploads from clients. It seems to me that we want to noisily fail there so that the problem with the leader can be investigated, right?